src/Ucommerce.Avenue.Umbraco/Controllers/Constants.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace Ucommerce.Avenue.Umbraco.Controllers
+{
+    public class Constants
+    {
+        public static readonly Guid CatalogGuid = new Guid("55BEFD9E-EFE8-6475-9BFC-C2A37349C708");
```

Comment:
Can I ask why we have this hardcoded?

Reply:
Certainly. In the case where we have two active ProductCatalogs, it was picking one indeterministically. I wanted to fixate it.
However, I find it much easier to just uncheck "Display On Site" for all but one and then run the indexing.
Thanks for raising the question.